src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,25 @@
+// TODO: move to SDS
+import React, { forwardRef } from "react";
+import "./styles.scss";
+
+interface TooltipProps {
+  isVisible: boolean;
+  children: React.ReactNode;
+  className?: string;
+}
+
+export const Tooltip = forwardRef<HTMLDivElement, TooltipProps>(
+  (props, ref) => {
+    const { isVisible, children, className } = props;
+
+    return (
+      <div
+        className={`Tooltip ${className ?? ""}`}
```

Comment:
very NIT: using `??` implies you want to use `className` if it is equal to `0` or `""`, but I think you probably want `className || ""` here

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
I think it's OK to allow a `0` string in this case. Thanks for pointing it out! 👍